app/redux/ducks/translations.js

```diff
@@ -0,0 +1,63 @@
+import apiClient from 'panoptes-client/lib/api-client';
+import counterpart from 'counterpart';
+
+counterpart.setFallbackLocale('en');
+// Actions
+const ERROR = 'pfe/translations/ERROR';
+const LOAD = 'pfe/translations/LOAD';
+const SET_LOCALE = 'pfe/translations/SET_LOCALE';
+const SET_STRINGS = 'pfe/translations/SET_STRINGS';
+
+const initialState = {
+  locale: 'en',
```

Comment:
Should this be `counterpart.getLocale()` rather than `en`?

Reply:
I think it should. As with any other hard-coded `'en'` string, right?

Reply:
I think so. I'm not sure if this should be set to the default counterpart locale, or use `getLocale()` in which case initial state can be changed by `counterpart.setLocale()`.

Reply:
Would setting it to the default counterpart locale be a hurdle for allowing researchers to have projects where the default is different from `en`, like this https://www.zooniverse.org/projects/stefanodf/occhio-alle-penne?

Reply:
Good question. Does this PR update the locale when the project loads? It should change to the project's primary language.

Reply:
My reading of the counterpart docs is that counterpart's default locale is always English, regardless of the browser's locale.

Reply:
[I think you're right](https://github.com/martinandert/react-translate-component#usage)
>Whenever the user selects a different locale from the drop-down, we correspondingly set the new drop-down's value as locale in the Counterpart library, which in turn triggers an event that our (soon to be integrated) Translate component listens to. As initially active value for the select element we specify Counterpart's current locale ("en" by default).

Reply:
My comment was wrong: when a project loads, we should respect the user's preferred locale, currently set via a query param. If that's not present, use the project's locale (which is always `en` at the moment, regardless of the actual text stored with the resource.)

Reply:
Either way, I don't think that has anything to do with the initial Redux state, because nothing has loaded when that is set. I think.